addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"account_account_template_4421","Prepayments from Service Receivers",4421,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_3","False"
+"account_account_template_4430","Debts to Suppliers for Goods and Services",4430,"account.data_account_type_payable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_4","True"
+"account_account_template_4431","Other Debts to Suppliers",4431,"account.data_account_type_payable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_4","True"
+"account_account_template_4470","Obligations of Profit Tax",4470,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_8","False"
+"account_account_template_4471","Other Obligations",4471,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_8","False"
+"account_account_template_4480","Payable Salary",4480,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_9","False"
+"account_account_template_4481","Payable Personal Income Tax",4481,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_9","False"
+"account_account_template_4482","Contributions of Payable Social Security",4482,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_9","False"
+"account_account_template_4483","Contributions of Payable Guarantee Fund",4483,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_9","False"
+"account_account_template_4485","Leave Liabilities",4485,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_9","False"
+"account_account_template_4486","Contributions of Payable Compulsory Health Insurance",4486,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_9","False"
+"account_account_template_4490","Payable Dividends",4490,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_10","False"
+"account_account_template_4491","Payable Bonuses",4491,"account.data_account_type_current_liabilities","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_10","False"
+"account_account_template_4492","Payable Value-Added Tax",4492,"account.data_account_type_payable","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_10","True"
```

Comment:
```suggestion
"account_account_template_4492","Payable Value-Added Tax",4492,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_g_2_10","True"
```

Reply:
@oerp-odoo , I'm sorry, but while writing the change suggestion I must have had a wrong type copied.
`4492` should be `data_account_type_current_liabilities` instead of `data_account_type_current_assets`. My bad...  😄